libs/components/tabs/testing/src/tabs/tabset-fixture.spec.ts

```diff
@@ -164,11 +164,16 @@ describe('Tabset fixture', () => {
         .nativeElement
     ).toHaveCssClass('sky-btn-tab-selected');
 
-    let errorMessage: string;
+    let errorMessage: string | undefined;
 
     try {
```

Comment:
This whole section can be refactored with something like this. Same for the other instances.

```
await expectAsync(tabset.clickTab(100)).toBeRejectedWith('There is no tab at index 100.');
```